Lib/sysconfig.py

```diff
@@ -84,9 +85,12 @@
 _SCHEME_KEYS = ('stdlib', 'platstdlib', 'purelib', 'platlib', 'include',
                 'scripts', 'data')
 
- # FIXME don't rely on sys.version here, its format is an implementation detail
- # of CPython, use sys.version_info or sys.hexversion
-_PY_VERSION = sys.version.split()[0]
+releaselevel_serial = ''
+if sys.version_info[3] in ('alpha', 'beta'):
+    releaselevel_serial = sys.version_info[3][0] + str(sys.version_info[4])
+elif sys.version_info[3] == 'release':
+    releaselevel_serial = 'rc' + str(sys.version_info[4])
+_PY_VERSION = '%d.%d.%d%s' % (*sys.version_info[:3], releaselevel_serial)
```

Comment:
This doesn't seem to address my concern in the bpo discussion, namely, the pre-release indicator is no longer in `py_version`.  For example, at the moment, if you build the 3.7 branch from its current HEAD, the output of `python -M sysconfig' includes:

`	py_version = "3.7.1+"
`
But with the PR as it stands, the output is now:

`	py_version = "3.7.1"`

And to make life interesting, the `platform` module still returns these values:

```
>>> platform.python_version()
'3.7.1+'
>>> platform.python_version_tuple()
('3', '7', '1+')

```

Sigh!  There are reasons why this issue is still open; it's not really obvious what the right answers are here.  Because it's hard to know where `py_version` might be being used in the wild, we should be cautious about changing its behavior.  I think we should probably discuss this back on the b-p-o issue.  Perhaps you could summarize things there?

Reply:
I posted on the bpo issue. If I understand correctly, I don't need to change something in this PR for the time being. After we've discussed it more thoroughly on the issue tracker, I can implement the requested changes, if we decide to move forward with this PR. Sounds OK?